src/ViewModels/WebMapServiceItemViewModel.js

```diff
@@ -108,8 +108,8 @@ var WebMapServiceItemViewModel = function(application) {
     delete this.__knockoutObservables.dataUrlType;
     knockout.defineProperty(this, 'dataUrlType', {
         get : function() {
-            if (defined(this._dataUrlType)) {
-                return this._dataUrlType;
+            if (defined(this._dataUrl)) {
+                return 'direct';
```

Comment:
Let's say I have a WMS and a WFS, but it's not a Geoserver serving both roles, it's two separate servers.  My `url` property might be `http://wms.made.up.nicta.com.au/fancyServer` and my `dataUrl` would be `http://wfs.made.up.nicta.com.au/someOtherServer`.  But that's not a direct URL.  It's a WFS URL that needs to have the layer name and such added.  This scenario would be impossible after your change.

Sure, it might not be an important use-case, but it's also easy to support (as was supported until this change).  In general, properties that are settable in a `CatalogItemViewModel` (such as `dataUrlType`) should also be settable in derived classes like this one, even if there is a reasonable default.